lib/AST/ASTScopeCreation.cpp

```diff
@@ -1346,8 +1346,6 @@ void EnumElementScope::expandAScopeThatDoesNotCreateANewInsertionPoint(
   if (auto *pl = decl->getParameterList())
     scopeCreator.constructExpandAndInsertUncheckable<ParameterListScope>(
         this, pl, nullptr);
-  // might contain a closure
-  scopeCreator.addToScopeTree(decl->getRawValueExpr(), this);
```

Comment:
This is suspect; invalid code might still contain a closure here. (Or if we ever allow constant expressions in enum raw values instead of just literals, but then most of this will change.)

Reply:
Spoke with @davidungar about this one.  It's always going to be a `LiteralExpr`, and we make sure that strings are never interpolated in Parse, so the invariant this was checking is prevented quasi-structurally.  That last case makes removing this not a fantastic idea - and I had only dumped it because it was one of the red herrings I found prevented the stdlib deserialization test from failing and because for now it won't actually do anything.

I'm going to change the comment to be a bit more helpful then.